<launch>

  <node name="auto_calibration_hsv" pkg="object_color_detector" type="auto_calibration_hsv.py" output="screen">
    <!-- load configuration -->
    <param name="filename"  value="$(find object_color_detector)/config/vision_config.yaml" />
  </node>

</launch>
